5.6/startup.sh

```diff
@@ -59,6 +59,11 @@ render_tmpl "$HOME_DIR/.acquia/cloudapi.conf"
 # Terminus authentication
 [[ "$SECRET_TERMINUS_TOKEN" ]] && terminus_login
 
+# Platform CLI Token
+if [[ -n "$SECRET_PLATFORMSH_CLI_TOKEN" ]]; then
+    PLATFORMSH_CLI_TOKEN=$SECRET_PLATFORMSH_CLI_TOKEN
```

Comment:
I had the same issue with the Blackfire API keys and left it alone for now. 
Since this is the second time this comes up, we should add a generic function that will transform/copy `SECRET_VAR` into `VAR` at `cli` startup.